### DISCUSSION TITLE
Disable notifications for a single container

### DISCUSSION THREAD

**Discussion by demonbane** [containrrr/watchtower#1957] [Notification Templates]
Is there any way to disable notifications for a single container while continuing to do everything else (including update that container)? The one in question gets updated upstream daily just in case, so I get a notification each and every day and I find that I now just ignore them. But this means that if I get a notification for a less frequently updated project I can often miss it since I'm just disabling these notifications by default.

**Comment by stanthewizzard** [comment]
The same
I need to disable notification for given containers
Thans
